docs/content.zh/docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -574,7 +574,7 @@ You can customize the Flink image in several ways:
 
     $ echo "
     # enable an optional library
-    ln -fs /opt/flink/opt/flink-queryable-state-runtime-*.jar /opt/flink/lib/
+    ln -fs /opt/flink/opt/flink-sql-gateway-*.jar /opt/flink/lib/
```

Comment:
Why the sql-gateway jar is needed ?

Reply:
I think here the purpose is to show how to enable an optional library with docker. It doesn't matter which library is brought in. Thus, if we don't want queryable state to appear in the documentation, any other library should be fine.